fea-rs/src/compile/lookups/helpers.rs

```diff
@@ -59,10 +59,10 @@ impl ClassDefBuilder2 {
     pub(crate) fn build(self) -> (ClassDef, HashMap<GlyphClass, u16>) {
```

Comment:
Where we deliberately match python code lets add a permalink to the python equivalent, I think that's true here and maybe elsewhere in this code?

For example,  https://github.com/googlefonts/fontations/pull/685/files does this.